src/Calculator.tsx

```diff
@@ -65,29 +65,29 @@ class Calculator extends React.Component {
         return (
             <div>
               <h1>Confusion Matrix</h1>
-              <table border="1">
+              <table style={{borderWidth: 1}}>
```

Comment:
> src/Calculator.tsx:65:22 - error TS2322: Type '{ children: Element[]; border: string; }' is not assignable to type 'DetailedHTMLProps<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement>'.
>   Property 'border' does not exist on type 'DetailedHTMLProps<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement>'.

`border` is a deprecated property; React is telling you to use `style` instead. It's not really too big of a deal but you might as well listen.